.github/workflows/coverage-report.yml

```diff
@@ -0,0 +1,41 @@
+name: 'coverage-report'
+on:
+  push:
+    branches:
+      - main
+      - coverage # remove before merge to main
```

Comment:
Looks like you used wrong branch name and we don't able to check the result

Reply:
This commit should fix this
[chore(ci): change test branch name for coverage report workflow](https://github.com/fingerprintjs/fingerprint-pro-server-api-php-sdk/pull/103/commits/87e0fac7e8be29b8d2f72557f400504c94ffbe74)